### PR TITLE
[2.0] 1488508: Restricted upstream info migration to master pools

### PR DESCRIPTION
**Pull request by Ceiu**
- The per-org migration task no longer migrates upstream subscription
  info to all related pools, but instead only updates the master pool
  for a given subscription

### CODE REVIEW

**Comment by Ceiu** [comment]
This is a speculative fix without a proper reproducer (at present). Currently awaiting word back from a community user for validation.

**Comment by Klaas-** [comment]
I've tested the fix and it works - it no longer creates the certificate_ids for derived products

**Comment by Klaas-** [comment]
its bz  1488508 though

**Comment by Ceiu** [comment]
For reviewers:
Before this gets merged, I'd like to add another task to do some cleanup on existing databases if the migration task appears to have been run already. Doing so safely is a tad troublesome, though, so when the commit is added, please give that bit a lot of attention.

**Comment by Ceiu** [comment]
@candlepin-pull-request-bot retest this please

**Comment by Ceiu** [comment]
@candlepin-pull-request-bot retest this please

**Comment by Ceiu** [comment]
@candlepin-pull-request-bot retest this please

**Comment by wottop** [comment]
performance ok

**Comment by wottop** [comment]
Ran against db with incorrect non-master values. The result was correct.

ack when the space is gone.

**Comment by Ceiu** [comment]
@wottop done.

File: server/src/main/resources/db/changelog/changelog-testing.xml [hunk 1]
```diff
@@ -2028,7 +2028,7 @@
     </changeSet>
     <changeSet author="awood" id="1413225753032-290">
         <comment>
-             Add the default quartz lock columns.
+             Add the default quartz lock columns. 
         </comment>
```
[the remark this answers is not in the document]
Comment (Ceiu) [reply server/src/main/resources/db/changelog/changelog-testing.xml]: Done.

**Pull request merged**